components/Home/SelectedCategories.vue

```diff
@@ -1,8 +1,8 @@
 <template>
   <div
     v-if="selectedMenuItems.length > 0"
-    class="flex flex-row p-2 flex-wrap bg-white shadow-md pointer-events-auto rounded-xl max-w-xl"
-    style="min-width: 64px"
+    class="list-of-categories-selected flex flex-row p-2 flex-wrap bg-white shadow-md pointer-events-auto rounded-xl max-w-sm min-w-xs"
+    :class="{ 'scroll-pr-2': selectedMenuItems.length > 1 }"
```

Comment:
To check

Reply:
removed unused class, conditionnal class works fine